packages/backend-lib/prisma/schema.prisma

```diff
@@ -203,6 +203,7 @@ model DefaultEmailProvider {
 
   emailProvider   EmailProvider @relation(fields: [emailProviderId], references: [id], onDelete: Cascade)
   emailProviderId String        @db.Uuid
+  fromAddress     String
```

Comment:
We need to make this an optional field, so that it doesn't break in prod.

Reply:
Note that you can use the following steps to replace the prisma schema without creating an additional migration.
1. Delete the migration subdirectory.
2. `yarn admin prisma migrate reset`
3. `yarn admin prisma migrate dev --name='add from address'`